pkg/services/sqlstore/sqlstore.go

```diff
@@ -110,17 +94,40 @@ func (ss *SQLStore) Init() error {
 	ss.addAlertNotificationUidByIdHandler()
 	ss.addPreferencesQueryAndCommandHandlers()
 
-	if err := ss.Reset(); err != nil {
-		return err
+	// if err := ss.Reset(); err != nil {
+	// 	return nil, err
+	// }
+	// // Make sure the changes are synced, so they get shared with eventual other DB connections
+	// // XXX: Why is this only relevant when not skipping migrations?
+	// if !ss.dbCfg.SkipMigrations {
+	// 	if err := ss.Sync(); err != nil {
+	// 		return nil, err
+	// 	}
+	// }
```

Comment:
Temporary hack. This needs to run after `Migrate`. `Migrate` is called by pkg/server/server.go in `init()`. Not sure what the best solution would be. Replace Migrate with `Run/start` method and do both migration, reset and sync or call Migrate first, then Reset followed by Sync?